cmd/lk/utils.go

```diff
@@ -142,6 +142,14 @@ func extractFlagOrArg(c *cli.Command, flag string) (string, error) {
 	return value, nil
 }
 
+func extractFlagAsBool(c *cli.Command, flag string) (bool, error) {
+	if c.IsSet(flag) {
+		return c.Bool(flag), nil
+	}
+
+	return false, nil
+}
```

Comment:
I don't see anything that could possibly error here, so it doesn't need to return one